.github/workflows/goreportcard.yaml

```diff
@@ -0,0 +1,51 @@
+name: Go Report Card
+
+on:
+  push:
+    branches:
+      - master
+      - main
+  pull_request:
+
+jobs:
+  goreportcard:
+    strategy:
+      matrix:
+        go: ['1.22']
+        os: [ubuntu-latest, macos-latest, windows-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+    - name: Setup Go
+      uses: actions/setup-go@v5
+      with:
+        go-version: ${{ matrix.go }}
+        cache: false
+    - name: Checkout gojp/goreportcard repo
+      uses: actions/checkout@v4
+      with:
+        repository: gojp/goreportcard
+        path: goreportcard
+    - name: Install goreportcard-cli
+      # goreportcard-cli requires the following linters:
+      #  1. gometalinter
+      #  2. golint
+      #  3. gocyclo
+      #  4. ineffassign
+      #  5. misspell
+      # among which, the linter gometalinter is deprecated. However, goreportcard repo has a vendor version of it.
+      # Hence installing from the repo instead of `go install`. Refer https://github.com/gojp/goreportcard/issues/301
+      run: |
+        cd goreportcard
+
+        # Install prerequisite linter binaries
+        make install
```

Comment:
Maybe the comment is unclear. Is it this step that install the 5 listed linters?

Reply:
Yes, it installs the 5 listed linters. Updated the comment. Ideally, it should install the `goreportcard-cli` binary along with prerequisites but it isn't. Maybe I'll add a PR there.

Reply:
Yes, is was surprised you add to install them all, golangci-lint sounds a better option. But I was unsure about your exact needs,so I didn't report about it

Reply:
I did try installing `golangci-lint` in this step to see if it is fetching all the required linters. The only issue is `gometalinter` which is public-archived and doesn't get downloaded. `goreportcard` however still uses it, so it has a vendored version in the repo.

This was discussed in https://github.com/gojp/goreportcard/issues/301

Reply:
It's indeed strange because gometalinter is indeed deprecated 😅

Reply:
> I did try installing `golangci-lint` in this step to see if it is fetching all the required linters. The only issue is `gometalinter` which is public-archived and doesn't get downloaded. `goreportcard` however still uses it, so it has a vendored version in the repo.
> 
> This was discussed in https://github.com/gojp/goreportcard/issues/301

I opened https://github.com/gojp/goreportcard/issues/425 for records